RELEASES.md

```diff
@@ -0,0 +1,198 @@
+
+### Swift for TensorFlow Deep Learning Library
+
+* The `Layer` protocol's `applied(to:in:)` method has been renamed to `call(_:)`.
+  `Layer`s are now "callable" like functions, e.g. `layer(input)`.
+  * **Note: this is experimental functionality that is currently [being proposed
+    through Swift
+    Evolution](https://github.com/apple/swift-evolution/blob/master/proposals/0253-callable.md).**
+    Expect potential changes.
+* The `context` argument has been removed from `Layer`'s `applied(to:)` method.
+  Instead, contexts are now thread-local. ([swift-apis#87](https://github.com/tensorflow/swift-apis/pull/87))
+  * Use `Context.local` to access the current thread-local context.
+  * Note: layers like `BatchNorm` and `Dropout` check `Context.local` to
+    determine whether the current learning phase is training or inference. **Be
+    sure to set the context learning phase to `.training` before running a
+    training loop.**
+  * Use `withContext(_:_:)` and `withLearningPhase(_:_:)` to call a closure
+    under a temporary context or learning phase, respectively.
+* A `RNNCell` protocol has been added, generalizing simple RNNs, LSTMs, and
+  GRUs. ([swift-apis#80](https://github.com/tensorflow/swift-apis/pull/80),
+  [swift-apis#86](https://github.com/tensorflow/swift-apis/pull/86))
+* New layers have been added.
+  * `Conv1D`, `MaxPool1D`, `AvgPool1D`.
+    ([swift-apis#57](https://github.com/tensorflow/swift-apis/pull/57))
+  * `UpSampling1D`.
+    ([swift-apis#61](https://github.com/tensorflow/swift-apis/pull/61))
+  * `TransposedConv2D`.
+    ([swift-apis#64](https://github.com/tensorflow/swift-apis/pull/64))
+  * `GlobalAveragePooling1D`, `GlobalAveragePooling2D`,
+    `GlobalAveragePooling3D`.
+    ([swift-apis#66](https://github.com/tensorflow/swift-apis/pull/66),
+    [swift-apis#65](https://github.com/tensorflow/swift-apis/pull/65),
+    [swift-apis#72](https://github.com/tensorflow/swift-apis/pull/72))
+* Optimizer stored properties (e.g. `learningRate`) are now mutable.
+  ([swift-apis#81](https://github.com/tensorflow/swift-apis/pull/81))
+
+### Automatic Differentiation
+
+* `Array` now conforms to `Differentiable`.
+  ([apple/swift#23183](https://github.com/apple/swift/pull/23183))
+* The `@differentiating` attribute now works when the derivative function has a
+  generic context that is more constrained than the original function's generic
+  context. ([apple/swift#23384](https://github.com/apple/swift/pull/23384))
+* The `@differentiating` attribute now accepts a `wrt` differentiation parameter
+  list, just like the `@differentiable` attribute.
+  ([apple/swift#23370](https://github.com/apple/swift/pull/23370))
+* The error `function is differentiable only with respect to a smaller subset of
+  arguments` is now obsolete.
+  ([apple/swift#23887](https://github.com/apple/swift/pull/23887))
+* A differentiation-related memory leak has been fixed.
+  ([apple/swift#24165](https://github.com/apple/swift/pull/24165))
+
```

Comment:
[TF-449](https://bugs.swift.org/browse/TF-449) should be mentioned since the fix is a bit more tricky and it won't make it to the release build.

Reply:
Done. I verified that TF-449 occurs using the v0.2 toolchain, so I filed it as a known issue under v0.2.